src/fides/api/util/consent_util.py

```diff
@@ -57,12 +59,17 @@ def filter_privacy_preferences_for_propagation(
     system: Optional[System], privacy_preferences: List[PrivacyPreferenceHistory]
 ) -> List[PrivacyPreferenceHistory]:
     """Filter privacy preferences on a privacy request to just the ones that should be considered for third party
-    consent propagation"""
+    consent propagation.
+
+    Only applies to preferences saved for privacy notices here, not against individual TCF components.
+    """
 
     propagatable_preferences: List[PrivacyPreferenceHistory] = [
         pref
         for pref in privacy_preferences
-        if pref.privacy_notice_history.enforcement_level == EnforcementLevel.system_wide
+        if pref.privacy_notice_history
+        and pref.privacy_notice_history.enforcement_level
+        == EnforcementLevel.system_wide
```

Comment:
Putting some extra checks here just to be comprehensive.  Currently, consent is only propagated to backend systems for notices, not TCF components.  Any privacy preference that hits here should have a privacy notice history attribute.  However, now privacy preferences are not required to have privacy notice histories, so here I'm verifying that one exists. 

Further work would need to be done if/when we get to a point where saving TCF preferences are propagated via API to third party systems.